### PR TITLE
[VEN-2550] Add FRAX and sFRAX Markets

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -238,7 +238,10 @@ const config: HardhatUserConfig = {
       url: process.env.ARCHIVE_NODE_sepolia || "https://ethereum-sepolia.blockpi.network/v1/rpc/public",
       chainId: 11155111,
       live: true,
-      accounts: process.env.DEPLOYER_PRIVATE_KEY ? [`0x${process.env.DEPLOYER_PRIVATE_KEY}`] : [],
+      // accounts: process.env.DEPLOYER_PRIVATE_KEY ? [`0x${process.env.DEPLOYER_PRIVATE_KEY}`] : [],
```
Comment (chechu) [inline hardhat.config.ts:241]: Is this change needed?

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -2628,6 +2678,18 @@ export const globalConfig: NetworkConfig = {
...
+            markets: ["FRAX"],
+            supplySpeeds: ["1481481481481481"], // 960 XVS over 90 days (648000 blocks)
+            borrowSpeeds: ["2222222222222222"], // 1440 CRV over 90 days (648000 blocks)
+          },
+          {
+            asset: "XVS",
+            markets: ["sFRAX"],
+            supplySpeeds: ["2222222222222222"], // 1440 CRV over 90 days (648000 blocks)
+            borrowSpeeds: ["1481481481481481"], // 960 XVS over 90 days (648000 blocks)
+          },
```
Comment (chechu) [inline helpers/deploymentConfig.ts:2692]: This is going to deploy two new RewardsDistributor contract, but I think we could reuse the already deployed RewardsDistributor with XVS as a reward token and the Core pool as comptroller (`RewardsDistributor_Core_0`)

To do that, I would:
* move this config to the object at line 2669
* ensure no new RewardsDistributor contracts are deployed (remove the deployment files for RewardsDistributor 2 and 3)
* in the VIP, only add the command `setRewardTokenSpeeds`

do you think it would work?

File: helpers/deploymentConfig.ts [hunk 3]
```diff
@@ -2616,11 +2666,31 @@ export const globalConfig: NetworkConfig = {
           //          81 XVS for Borrowers
           // crvUSD:  24 XVS for Suppliers
           //          36 XVS for Borrowers
+          // FRAX:    960 XVS for Suppliers
+          //          1440 XVS for Borrowers
+          // sFRAX:   1440 XVS for Suppliers
+          //          2160 XVS for Borrowers
```
Comment (chechu) [inline helpers/deploymentConfig.ts:2672]:
```suggestion
          //          960 XVS for Borrowers
```

**Review by chechu — APPROVED** [review]

**Review by GitGuru7 — APPROVED** [review]
